bot/bot.go

```diff
@@ -136,101 +144,126 @@ func askMember(s *discordgo.Session, owner string, message string) error {
-				origin, err := utils.GetChannel(channs, params[2])
+			} else if numParams == 3 {
+				log.Println("Received move command with 3 parameters on " + guild.Name + " , ID: " + guild.ID + " , by :" + m.Author.ID)
+				origin, err := utils.GetChannel(channs, params[1])
 				if err != nil {
-					s.ChannelMessageSend(m.ChannelID, fmt.Sprintf(messages[langg]["CantFindChannel"], params[2]))
+					_, _ = s.ChannelMessageSend(m.ChannelID, fmt.Sprintf(messages[lang]["CantFindChannel"], params[1]))
+					return
 				}
-				destination, err := utils.GetChannel(channs, params[3])
+
+				destination, err := utils.GetChannel(channs, params[2])
 				if err != nil {
-					s.ChannelMessageSend(m.ChannelID, fmt.Sprintf(messages[langg]["CantFindChannel"], params[3]))
+					_, _ = s.ChannelMessageSend(m.ChannelID, fmt.Sprintf(messages[lang]["CantFindChannel"], params[2]))
+					return
 				}
+
 				if !utils.CheckPermissions(s, origin, m.Author.ID, discordgo.PermissionVoiceMoveMembers) {
-					s.ChannelMessageSend(m.ChannelID, messages[langg]["NoPermissionsOrigin"])
+					_, _ = s.ChannelMessageSend(m.ChannelID, messages[lang]["NoPermissionsOrigin"])
 					return
 				}
+
 				if !utils.CheckPermissions(s, destination, m.Author.ID, discordgo.PermissionVoiceMoveMembers) {
-					s.ChannelMessageSend(m.ChannelID, messages[langg]["NoPermissionsDestination"])
+					_, _ = s.ChannelMessageSend(m.ChannelID, messages[lang]["NoPermissionsDestination"])
 					return
 				}
+
 				num, err := mover.MoveOriginDestination(s, <-workerschann, m, guild, botPrefix, origin, destination)
 				if err != nil {
-					s.ChannelMessageSend(m.ChannelID, fmt.Sprintf(messages[langg]["JustMoved"], err.Error()))
+					_, _ = s.ChannelMessageSend(m.ChannelID, fmt.Sprintf(messages[lang]["JustMoved"], err.Error()))
 					log.Println(err.Error())
 					return
 				}
-				s.ChannelMessageSend(m.ChannelID, fmt.Sprintf(messages[langg]["JustMoved"], num))
+
+				_, _ = s.ChannelMessageSend(m.ChannelID, fmt.Sprintf(messages[lang]["JustMoved"], num))
 				go bumpStatistics(num, s, conn)
-				return
+			} else {
+				log.Println("Received move command with " + strconv.Itoa(numParams) + " parameter(s) (help message) on " + guild.Name + " , ID: " + guild.ID)
+				_, _ = s.ChannelMessageSend(m.ChannelID, mover.MoveHelper(channs, messages[lang]["MoveHelper"], botPrefix))
 			}
-			log.Println("Sending help message on " + guild.Name + " , ID: " + guild.ID)
-			s.ChannelMessageSend(m.ChannelID, mover.MoveHelper(channs, messages[langg]["MoveHelper"], botPrefix))
-		} else {
-			s.ChannelMessageSend(m.ChannelID, fmt.Sprintf(messages[langg]["EhhMessage"], m.Author.Mention(), m.Content, botPrefix))
+
+		case "help":
+			_, _ = s.ChannelMessageSend(m.ChannelID, fmt.Sprintf(messages[lang]["HelpMessage"], botPrefix, botPrefix))
+
+		case "lang":
+			if numParams == 2 {
+				chosenLang := utils.SelectLang(params[2])
```

Comment:
This is causing a panic while testing language selection "panic: runtime error: index out of range"

Reply:
fixed by commit 3914aa3f27c0e11a4667d1dda0fe0fceb60e9799